### ISSUE TITLE
Bug: DDPG update_lr method calls sys.exit()

### ISSUE THREAD

**Issue by AntonJoha** [AntonJoha/neural_network_models#41]
## Issue

In `RL/cddpg_agent.py`, the `update_lr` method incorrectly calls `sys.exit()` instead of updating learning rates:

```python
def update_lr(self, count):
    sys.exit("Learning rate is to be updated")  # Terminates program!
```

## Impact
Program crashes when learning rates are attempted to be updated.

## Expected Behavior
Should update learning rate for the actor and critic optimizers based on training progress.

## Fix
Replace `sys.exit("Learning rate is to be updated")` with actual learning rate update logic.
